apps/dolly-frontend/src/main/js/src/components/fagsystem/tpsf/form/personinformasjon/Personinformasjon.js

```diff
@@ -48,6 +49,10 @@ const vergemaalPath = ['tpsf.vergemaal']
 const fullmaktPath = ['pdldata.person.fullmakt']
 
 const sikkerhetstiltakPaths = [
+	'pdldata.person.sikkerhetstiltak[0].tiltakstype',
```

Comment:
Vi trenger vel bare én sikkerhetstiltakPath nå som de ligger i samme objekt? `pdldata.person.sikkerhetstiltak` 😊 
Og alle på tpsf trenger vi vel ikke?

Reply:
Ja sant, gikk på autopilot der ser jeg 😅  Tror vi trenger tpsf for å støtte gjenopprett av gamle bestillinger med tpsf format 🤓

Reply:
Hmm, ja det er sant. Det har jeg vel egentlig ikke gjort selv når jeg har laget andre ting. Men det er vel begrenset hvor lenge vi kan støtte det også? 🤔

Reply:
Ja det vil jo ikke støttes så altfor lenge, men blir vel grei sak å fjerne alt av tpsf når den tid kommer 😄

Reply:
Sant sant! 😄

Reply:
Vet ikke om jeg misforstår, men fra frontend skal det ikke legges inn i tpsf-struktur på sikkerhetstiltak, telefonnr etc. Bakoverkompabilitet her skal ivaretas av backend. 😸

Reply:
Da har jeg misforstått tror jeg, ved gjenoppretting av gamle bestillinger som var sendt til tpsf så skal backend gjøre de om til ny struktur før det sendes til frontend? 😄